components/brave_wallet/brave_wallet_utils.cc

```diff
@@ -133,4 +137,48 @@ std::string Uint256ValueToHex(uint256_t input) {
   return out;
 }
 
+std::string GenerateMnemonic(const std::vector<uint8_t>& entropy) {
+  uint8_t* entropy_in = const_cast<uint8_t*>(entropy.data());
+  size_t size_in = entropy.size();
+
+  // If input entropy is empty, we generate a random 128 bits
+  std::vector<uint8_t> bytes(16);
```

Comment:
I suppose defaulting to 16 is fine, but if we instead do as suggested above, we would simply generate enough space for the requested number of bits OR default to 128.

Reply:
fixed in ca4d806d885cb83113b7163a9861ffdbfde28203